auth/shared-signal-receiver/tests/unit/handlers/request-handler.unit.test.ts

```diff
@@ -95,4 +114,102 @@ describe('requestHandler', () => {
     const result = await requestHandler(input);
     expect(result.statusCode).toBe(StatusCodes.BAD_REQUEST);
   });
+
+  it('return BAD_REQUEST 400 response for unsupported change type', async () => {
+    const verifyUsernameMock = verifyUsername as Mock;
+    verifyUsernameMock.mockResolvedValue(true);
+
+    const input = {
+      iss: 'https://identity.example.com',
+      jti: '123e4567-e89b-12d3-a456-426614174000',
+      iat: 1721126400,
+      aud: 'https://service.example.gov.uk',
```

Comment:
We could reduce repetition by creating a helper function that returns most of the common input fields